apps/src/applab/applab.js

```diff
@@ -1505,10 +1505,10 @@ Applab.showConfirmationDialog = function(config) {
 
   var buttons = document.createElement('div');
   ReactDOM.render(
-    React.createElement(DialogButtons, {
-      confirmText: commonMsg.dialogOK(),
-      cancelText: commonMsg.dialogCancel()
-    }),
+    <DialogButtons
+      confirmText={commonMsg.dialogOK()}
+      cancelText={commonMsg.dialogCancel()}
+    />,
```

Comment:
I find it easiest to search for component usage via grepping for `<DialogButtons` and I think it's more readable than React.createElement